projects/plugins/jetpack/_inc/client/state/recommendations/reducer.js

```diff
@@ -199,6 +201,23 @@ const siteDiscount = ( state = {}, action ) => {
 	}
 };
 
+const installing = ( state = {}, action ) => {
+	switch ( action.type ) {
+		case JETPACK_RECOMMENDATIONS_FEATURE_INSTALL_START:
+			return {
+				...state,
+				[ action.feature ]: true,
```

Comment:
Wondering if there's any benefit to checking that `action.feature` is an expected value here?

Reply:
I'm sorry, could you elaborate what you mean by this?

Reply:
That's funny, in this P2 post that was posted earlier today: pbOQVh-29D-p2, one of the things talked about was always handling errors or null values even if it extremely unlikely. If I take advice from that, I would think we should add it just to be safe. Especially because it's not an expensive check

Reply:
That is a very small thing though and not critical, I think I'll approve this PR and you can change that if you feel it is necessary 😄

Reply:
Added a check for this 👍